src/docs/product/integrations/aws-lambda/index.mdx

```diff
@@ -0,0 +1,99 @@
+
+</Alert>
+
+   
+1. Review the Lambda functions you want instrumented with Sentry. Lambda functions which are selected will get the Sentry layer and necessary environment variables automatically added when you press **Finish Setup**. Note that you can enable/disable Sentry on Lambda functions after installation as well.
+
+   ![](select_lambdas.png)
+
+1. Depending on how many Lambda functions you enable, you might need to wait a few minutes for this step to complete. If Sentry is able to update all you functions successfully, installation is considered finished. If there are errors, you will see a list of which Lambda functions failed along with a description on why they failed. 
+   
+### Troubleshooting
+
+#### Error `SentryRole already exists`
+
+The stack already exists on your account so you can go to the next step of adding your AWS account information.
+
+#### Error `Invalid existing layer`
+
+Your Lambda function has a layer that no longer exists. Please remove the problematic layer and then enable Sentry on the Lambda in the configuration view for the integration.
+
+#### Error `Please validate the CloudFormation stack was created successfully` 
+
+Sentry is unable to access your AWS account through your CloudFormation stack. This means either the stack has not been created or the external ID used doesn't match the one in your installation. You can go to the Parameters tab of the CloudFormation stack to get the value of the external ID and paste it into the form field.
+
+## Configure
+
+The integration configuration view allows you to do the following with your Lambda functions:
+
+* Enable Sentry
+* Disable Sentry
+* Update the Sentry Lambda Layer to the latest version
+
+In the table view, you will also see the version of the Lambda layer below the name.
+
+   ![](configure.png)
+   
+   
+## Uninstallation
+
+Note that when the integration is uninstalled, your Lambda functions will still be instrumented with Sentry and you will continue to receive errors. If you want to disable Sentry for all your Lambda functions, you should disable Sentry for each function before uninstalling.
+
+
+## How it Works
+
+When Sentry is added to a Lambda function, the following modifications are made to your Lambda functions:
+* The layer for Sentry is added to your Lambdas. Any existing layers are honored and the Sentry layer is appended to the end.
+
+   ![](layer.png)
+   
+* Sentry adds the following environment variables
+  * `NODE_OPTIONS`: This is to preload the `awslambda-auto` module which will automatically initialize Sentry
+  * `SENTRY_DSN`: This is set to the DSN of your project
+  * `SENTRY_TRACES_SAMPLE_RATE`: This sets the sampling rate for transactions
```

Comment:
We should explain that users can adjust this parameter to control the sample rate for each Lambda function. Or may be just point to perf docs for more info.